sdk/messaging/internal/uuid/uuid.go

```diff
@@ -0,0 +1,72 @@
+package uuid
```

Comment:
We already have a `uuid` package in the core `internal` module.

Reply:
Done.